pkg/storage/cacher.go

```diff
@@ -467,8 +467,8 @@ type cacheWatcher struct {
 
 func newCacheWatcher(resourceVersion uint64, initEvents []watchCacheEvent, filter FilterFunc, forget func(bool)) *cacheWatcher {
 	watcher := &cacheWatcher{
-		input:   make(chan watchCacheEvent, 10),
-		result:  make(chan watch.Event, 10),
+		input:   make(chan watchCacheEvent, 100),
```

Comment:
@xiang90 
> why 100, not 200, 500 or 1024?

No particular reason actually. I choose this value because 10 is too small, ant etcd use 100: 
https://github.com/coreos/etcd/blob/master/store/watcher_hub.go#L69

Do you guys have any reason about the 100?